### PR TITLE
SITES-7749: remove everything not relevant for email

### PR DESCRIPTION
**Pull request by Buuhuu**
This removes data attributes from components and metadata from the page that are not relevant for email, including:

- datalayer setup and instrumentation 
- data attributes utilised by javascript functionality
- seo links and robots tag
- pwa support
- config module*

(*) The config module only contained a service user mapping, which is conflicting with the one shipped by the wcm core components. However, it is not required anymore as service user authentication is not used by the core email components anymore.

Relates to #186

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html [hunk 1]
```diff
@@ -44,19 +41,15 @@
...
+         class="${!wcmmode.disabled ? 'cmp-tabs__tabpanel cmp-tabs__tabpanel--active' : ''}"></div>
     <table data-sly-test="${wcmmode.disabled}" role="presentation">
         <sly data-sly-repeat.item="${tabs.items}"
              data-sly-call="${itemTemplate.item @ item = item}"
              id="${item.id}-tabpanel-noedit"
              role="tabpanel"
              aria-labelledby="${item.id}-tab"
              tabindex="0"
-             class="${wcmmode.edit ? 'cmp-tabs__tabpanel cmp-tabs__tabpanel--active' : ''}"
-             data-cmp-hook-tabs="tabpanel"
```
Comment (bpauli) [inline content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html:58]: this data attribute is required by the panelcontainer.js

File: content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html [hunk 2]
```diff
@@ -34,8 +32,7 @@
             id="${tab.id}-tab"
             class="cmp-tabs__tab${isActive ? ' cmp-tabs__tab--active' : ''}"
             aria-controls="${tab.id}-tabpanel"
-            tabindex="${isActive ? '0' : '-1'}"
-            data-cmp-hook-tabs="tab">${tab.title}
```
Comment (bpauli) [inline content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html:38]: this data attribute is required by the panelcontainer.js

**Review by bpauli — CHANGES_REQUESTED** [review]

File: content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html (hunk 1 above)
Comment (Buuhuu) [reply content/src/content/jcr_root/apps/core/email/components/segmentation/v1/segmentation/segmentation.html:58]: right! not exactly in this place as the table is only rendered for wcmmode == disabled, but earlier it is required; as well as the `data-cmp-is`. The attributes on the enclosing `div` don't matter though as it is unwrapped in wcmmode == disabled anyway. So I removed the unnecessary conditions there as well and streamlined them in the whole file. See https://github.com/adobe/aem-core-email-components/pull/234/commits/3a9c51fb7b90983511ab68e2d315163b358062a2

File: content/src/content/jcr_root/apps/core/email/components/image/v1/image/image.html [hunk 3]
```diff
@@ -16,24 +16,19 @@
 <table data-sly-use.image="com.adobe.cq.email.core.components.models.Image"
        data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
        data-sly-test="${image.src}"
-       data-cmp-dmimage="${image.dmImage}"
```
[the remark this answers is not in the document]
Comment (Buuhuu) [reply content/src/content/jcr_root/apps/core/email/components/image/v1/image/image.html:19]: Not sure how we will implement #232 as we don't have javascript available in emails. Most likely we cannot rely on data attributes so I removed it.

**Review by bpauli — APPROVED** [review]

**Pull request merged**